[2025-04-06T11:23:43.041+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-06T11:23:43.052+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.create_weather_table manual__2025-04-06T11:23:38.337814+00:00 [queued]>
[2025-04-06T11:23:43.058+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.create_weather_table manual__2025-04-06T11:23:38.337814+00:00 [queued]>
[2025-04-06T11:23:43.059+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-04-06T11:23:43.069+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): create_weather_table> on 2025-04-06 11:23:38.337814+00:00
[2025-04-06T11:23:43.080+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=219156) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-06T11:23:43.081+0000] {standard_task_runner.py:72} INFO - Started process 219172 to run task
[2025-04-06T11:23:43.080+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'create_weather_table', 'manual__2025-04-06T11:23:38.337814+00:00', '--job-id', '732', '--raw', '--subdir', 'DAGS_FOLDER/weatherAPIDAG.py', '--cfg-path', '/tmp/tmp58pze5b5']
[2025-04-06T11:23:43.082+0000] {standard_task_runner.py:105} INFO - Job 732: Subtask create_weather_table
[2025-04-06T11:23:43.115+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_api_dag.create_weather_table manual__2025-04-06T11:23:38.337814+00:00 [running]> on host ab13fcc6a433
[2025-04-06T11:23:43.179+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='create_weather_table' AIRFLOW_CTX_EXECUTION_DATE='2025-04-06T11:23:38.337814+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-06T11:23:38.337814+00:00'
[2025-04-06T11:23:43.181+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-06T11:23:43.181+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-06T11:23:43.181+0000] {logging_mixin.py:190} INFO - Current task name:create_weather_table state:running start_date:2025-04-06 11:23:43.053006+00:00
[2025-04-06T11:23:43.182+0000] {logging_mixin.py:190} INFO - Dag name:weather_api_dag and current dag run status:running
[2025-04-06T11:23:43.182+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-06T11:23:43.189+0000] {base.py:84} INFO - Retrieving connection 'weather_***_conn'
[2025-04-06T11:23:43.196+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-06T11:23:43.203+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-06T11:23:43.203+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=create_weather_table, run_id=manual__2025-04-06T11:23:38.337814+00:00, execution_date=20250406T112338, start_date=20250406T112343, end_date=20250406T112343
[2025-04-06T11:23:43.210+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-06T11:23:43.210+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-06T11:23:43.210+0000] {logging_mixin.py:190} INFO - Dag name:weather_api_dag queued_at:2025-04-06 11:23:38.345412+00:00
[2025-04-06T11:23:43.211+0000] {logging_mixin.py:190} INFO - Task hostname:ab13fcc6a433 operator:PythonOperator
[2025-04-06T11:23:43.256+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-06T11:23:43.276+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-06T11:23:43.277+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
